### PR TITLE
Feature: Simplified the see more button on the tags widget

### LINKED ISSUE #11473: Feature: Replace See More string in Tags Home widget with a right facing chevron
### What feature or improvement do you think would benefit Files?

Please replace the string "View more" in Tags Home widget, with a GUI element like '...' otherwise it is confusing to mix readable content like "View more" which looks and feels like a users personal file, with other information.
Replace "View more" with just '...'

### Requirements

- Replace See More string in Tags Home widget with a right facing chevron in the container hader

### Files Version

2.4.40.0

### Windows Version

Win11 22H2 10.0.22621.1265 

### Comments

Tags Home See More

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #11473 

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [x] Did you check for accessibility? You can use Accessibility Insights for this.
- [x] Did you remove any strings from the en-us resource file?
   - [x] Did you search the solution to see if the string is still being used? 
- [x] Did you implement any design changes to an existing feature?
   - [x] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

**Screenshots (optional)**
*Before:*
Before_Dark Before_Light

*After*
After_Dark After_Light

### CODE REVIEW

File: src/Files.App/UserControls/Widgets/FileTagsWidget.xaml [hunk 1]
```diff
@@ -47,27 +47,43 @@
...
-					</StackPanel>
+					<Grid BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}" BorderThickness="0,0,0,1">
+						<Grid.ColumnDefinitions>
+							<ColumnDefinition Width="Auto" />
+							<ColumnDefinition Width="*" />
+							<ColumnDefinition Width="Auto" />
+						</Grid.ColumnDefinitions>
+
+						<StackPanel
+							Padding="12,8"
```
Comment (yaira2) [inline src/Files.App/UserControls/Widgets/FileTagsWidget.xaml:61]:
```suggestion
							Padding="12,8,12,8"
```

File: src/Files.App/UserControls/Widgets/FileTagsWidget.xaml [hunk 2]
```diff
@@ -47,27 +47,43 @@
...
+								Height="12"
+								Fill="{x:Bind Color, Mode=OneWay, Converter={StaticResource StringToBrushConverter}}" />
+							<TextBlock
+								Margin="0,-2,0,0"
+								FontWeight="SemiBold"
+								Text="{x:Bind Name, Mode=OneWay}" />
+						</StackPanel>
+
+						<!--  View More  -->
+						<HyperlinkButton
```
Comment (yaira2) [inline src/Files.App/UserControls/Widgets/FileTagsWidget.xaml:75]: Can you add an AutomationName for the screen reader? You might have to add back the `ViewMore` string to the resource file.

Comment (yaira2) [reply src/Files.App/UserControls/Widgets/FileTagsWidget.xaml:75]: > Did you check for accessibility? You can use Accessibility Insights for this.

This step would have failed 😅

Comment (ferrariofilippo) [reply src/Files.App/UserControls/Widgets/FileTagsWidget.xaml:75]: Done

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**